### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by maxme1**

### CODE REVIEW

File: amid/msd.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    -----
+    Data can be downloaded here:http://medicaldecathlon.com/
+    or here: https://msd-for-monai.s3-us-west-2.amazonaws.com/
+    or here: https://drive.google.com/drive/folders/1HqEgzS8BV2c7xYNrZdEAnrHk7osJJ--2/
+    Then, the folder with raw downloaded data should contain tar archive with data and masks
+    (`Task03_Liver.tar`).
+    """
+
+    _root: str = None
+    _task_to_name: dict = {
```
Comment (maxme1) [inline amid/msd.py:45]: do you really need this as a layer argument?

File: amid/msd.py [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+        with open_nii_gz_from_tar(tar_path, file_path) as nii_image:
+            # most CT/MRI scans are integer-valued, this will help us improve compression rates
+            return np.int16(nii_image.get_fdata())
+
+    def mask(_file):
+        tar_path, file_path = _file
+        if 'imagesTs' not in file_path:
+            with open_nii_gz_from_tar(tar_path, file_path.replace('images', 'labels')) as nii_image:
+                return np.uint8(nii_image.get_fdata())
+        return
```
Comment (maxme1) [inline amid/msd.py:105]: an empty return does nothing here anyway

File: amid/msd.py (hunk 1 above)
Comment (kurmukovai) [reply amid/msd.py:45]: is there any other way to define a field in Connectome dataset? Should we do it like this?
```python
class Dataset(Source):
    def _task_to_name():
        return dict(...)
```

Comment (maxme1) [reply amid/msd.py:45]: not really. I would use a global constant here

**Review by kurmukovai — APPROVED** [review]

**Pull request merged**